egui_glium/Cargo.toml

```diff
@@ -22,25 +22,22 @@ include = [
 all-features = true
 
 [dependencies]
-copypasta = "0.7"
 egui = { version = "0.14.0", path = "../egui", default-features = false, features = ["single_threaded"] }
+egui_for_winit = { version = "0.14.0", path = "../egui_for_winit" }
```

Comment:
There's no way to disable `copypasta` or `webbrowser` as a user of `egui_glium`, since they're on by default on a dependency.

Maybe if you set `default-features=false` on this line, and then add `default=[egui_for_winit/{copypasta,webbrowser}]`, that might work?

Reply:
Another note on this line is the version number, 0.14, this likely should be 0.1 or completely removed right? Pay is taken by default anyways.

Reply:
Are people likely to want to disable these crates? It seems like both are pretty nice to have, and small dependencies compared to winit.

If so, perhaps the features should be given proper names (that are not tied to the implementation), e.g. `clipboard` and `links` ?

Reply:
> Are people likely to want to disable these crates? It seems like both are pretty nice to have, and small dependencies compared to winit.

I like keeping dependencies low where needed, and try to disable whatever I don't use.
I think there are also situations where egui might be used on a device that doesn't have a webbrowser, or doesn't support clipboard operations (e.g. console) so it's worth supporting turning them off

> If so, perhaps the features should be given proper names (that are not tied to the implementation), e.g. `clipboard` and `links`?

Yeah that's probably a good idea

Reply:
Good point!